src/daos/Airtable/CanvasCoursesDao.ts

```diff
@@ -0,0 +1,57 @@
+import Airtable, { FieldSet, Records } from "airtable";
+import { AirtableBase } from "airtable/lib/airtable_base";
+
+class CanvasCoursesDao {
+  private api_key: string;
+  private base_id: string;
+  public airtable: AirtableBase;
+
+  constructor() {
+    if (!process.env.AT_API_KEY) {
```

Comment:
I really like how there checks for your setup here in the constructor. Maybe we make that a standard across the entire app?

Reply:
[Added this for the DAOs with locals to initialize](https://github.com/Lambda-School-Labs/labs-be/pull/14/commits/cfbc7d8903736c135c64e17cb17142e007d5a9b9)